### DISCUSSION TITLE
Can you add a function: Export A folder of the project, rather than the current can only export all the projects, but can not selectively export the project folder

### DISCUSSION THREAD

**Discussion by xwndo** [dani-garcia/vaultwarden#4897] [Ideas]
Sometimes you need to share backups, or do local batch editing, but you only need to modify some projects, not all, and you need to export only the projects in several folders, which you hope the developers can support.

**Comment by BlackDex** [comment]
I would suggest to use the Bitwarden CLI `serve` option which exposes an API to do such stuff.
